### PR TITLE
Rewrite config page

### LINKED ISSUE #2396: Replace config.yml example with a link to the default config file
It's out of date and doesn't document some options, we should add those and link to the canonical default config instead

### PR DESCRIPTION
**Pull request by evertiro**
Fixes https://github.com/Varying-Vagrant-Vagrants/VVV/issues/2396... I think.

Disclaimer: It seems Jekyll is another one of those things that don't like M1 processors. However, I didn't realize until I'd already reworked the page and decided that at least it gets the ball rolling. This PR needs testing before it's merged, and I'm sure half of it will be rewritten, but at least it's a start.

### CODE REVIEW

**Comment by Mte90** [comment]
To me is good but I think that is the case to update before the config.yml in VVV so we updated there too

**Comment by tomjn** [comment]
@evertiro you can just run it under rosetta, either by launching terminal.app using rosetta or via the CLI params with `arch -x86_64 `

File: docs/en-US/config.yml.md [hunk 1]
```diff
@@ -10,80 +10,185 @@ permalink: /docs/en-US/config/
 
 **Before VVV 3.2 the file was `vvv-config.yml` and `vvv-custom.yml` in the VVV root.**
 
-Here's the full default config file, with every key and option that VVV supports:
+Here's the full default config file:
 
 ```yaml
+---
+
```
Comment (tomjn) [inline docs/en-US/config.yml.md]: I was hoping to link to it rather than embed it, that way we don't have to keep going back and updating it.

Comment (tomjn) [reply docs/en-US/config.yml.md]: Also the default config doesn't contain all the keys and options anyway

Comment (evertiro) [reply docs/en-US/config.yml.md]: I'd noticed.... but we don't have any one file that actually lists _every_ option.

Comment (evertiro) [reply docs/en-US/config.yml.md]: And I did link to it... after the embed.

Comment (tomjn) [reply docs/en-US/config.yml.md]: more that the embed itself is problematic, people draw conclusions, it'd be better to expand the docs with the available options

Comment (evertiro) [reply docs/en-US/config.yml.md]: Ok, let's move away from it. I did actually add (at least most) of the missing options, but I'm also sure I missed a few.

**Review by Mte90 — DISMISSED** [review]

**Review by tomjn — APPROVED** [review]

**Pull request merged**